README.md

```diff
@@ -85,24 +68,28 @@ catch( Exception e )
     Debug.LogError( e );
 }
 ```
+In the above snippet, if an account does not exist a Wallet and key pair will be created.
 
-Calling `GetAccount` with the existing account index, will retrieve the account stored on the device.
+Calling `GetAccount` with the existing account index will retrieve the account stored on the device.
 
 ```csharp
 if( kinClient.HasAccount() )
-    account = kinClient.GetAccount( 0 );
```

Comment:
The method name was correct, in c# methods names are in CamelCase, unlike java where its "lowerCamelCase"